plugins/outputs/clarify/sample.conf

```diff
@@ -0,0 +1,12 @@
+## Credentials File (Oauth 2.0 from Clarify integration)
+credentials_file = "/path/to/clarify/credentials.json"
+
+## Clarify username password (Basic Auth from Clarify integration)
+username = "i-am-bob"
+password = "secret-password"
+
+## Timeout for Clarify operations
+# timeout = "20s"
+
+## Optional tags to be included when generating the unique ID for a signal in Clarify
+# id_tags = ['sensor']
```

Comment:
Commented out options should represent default values. This is not the case for `id_tags`.